vaults/README.md

```diff
@@ -0,0 +1,153 @@
+
+## Prerequisites
+
+To segregate responsibilities, we would create two distinguished namesapces 
+- `<OperatorKubernetesNameSpace>` Kubernetes namespace which will receive the vault operator
+- `<VaultKubernetesNameSpace>` Kubernetes namespace which will receive vault servers
+
+```shell
+kubectl create namespace <OperatorKubernetesNameSpace>
+kubectl create namespace <VaultKubernetesNameSpace>
+```
+
+## Deployment
+
+### [Vault Operator](https://banzaicloud.com/docs/bank-vaults/operator/)
+
+It will be used the Banzai Cloud Vault Operator to manage the deployment of Vault servers
+
+```shell
+helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com
+helm upgrade --namespace <OperatorKubernetesNameSpace> --install vault-operator banzaicloud-stable/vault-operator
+```
+
+### Configuration
+#### Vault instances
+
+To deploy Harshicorps Vault in your environment, you have to set configuration in `operator-deploy/vault-k8s-sample.yaml` 
+
+Change the `<VaultKubernetesNameSpace>` by the namespace where you want to deploy your Vault instance
+```helmyaml
+  unsealConfig:
+    options:
+      # The preFlightChecks flag enables unseal and root token storage tests
+      # This is true by default
+      preFlightChecks: true
+    kubernetes:
+      secretNamespace: <VaultKubernetesNameSpace>
+``` 
+
+Define and specify the rules for your [Vault policy](https://www.vaultproject.io/docs/concepts/policies)
+```helmyaml
+  externalConfig:
+    policies:
+      - name: allow_secrets
+        rules: path "secret/*" {
+          capabilities = ["create", "read", "update", "delete", "list"]
+          }
+      - name: tx_signer_demo
+        rules: path "secret/data/orchestrate-demo/keys/*" {
+          capabilities = ["create", "read", "update", "delete", "list"]
+          }
+```
+
+Define and specify the rules for your [Vault authentication](https://www.vaultproject.io/docs/concepts/auth)
+```helmyaml
+  externalConfig:
+    auth:
+      - type: kubernetes
+        roles:
+          # Allow every pod in the default namespace to use the secret kv store
+          - name: tx-signer
+            bound_service_account_names: ["tx-signer", "vault-secrets-webhook", "vault"]
+            bound_service_account_namespaces: ["operator", "vault-sandbox", "orchestrate-demo"]
+            policies: ["allow_secrets", "tx_signer_demo"]
+```
+
+Define and specify the rules for your [Vault Secrets Engines](https://www.vaultproject.io/docs/secrets)
+
+   !!! Note:
+    The Secrets Engines for Orchestrate is Key/Value version 2
+
+```helmyaml
+    secrets:
+      - path: secret
+        type: kv
+        description: General secrets.
+        options:
+          version: 2
+```
+
+#### Orchestrate environment
+
+In the `environments` directory, you need to set variables to connect the `tx-signer` to the harshicorp vault
+
+- `SECRET_STORE`: Value have to be `hashicorp` to connect to Harshicorp Vault instance.
```

Comment:
I understand what you mean but it appears a bit confusing, maybe it would be worthy to do a rephrase, such as:

"Secret storage type. Use `hashicorp` to connect use Harshicorp Vault instance"